nova/nova/views.py

```diff
@@ -71,4 +126,4 @@ def auth_tokens_coll(request):
         raise AuthenticationFailed()
 
     token, created = Token.objects.get_or_create(user=user)
-    return Response({'token': token.key, 'user_id': user.pk})
+    return Response({'token': token.key, 'user_id': user.pk}, status=status.HTTP_200_OK)
```

Comment:
I think the default status is 200, you don't need to pass it. If you added this part for the sake of explicitness ,then it's alright